### PR TITLE
Feature/xp preprod

### PR DESCRIPTION
**Pull request by dms1981**
Creates a preoproduction environment for Xhibit Portal, allocates range for HMCTS preproduction

### CODE REVIEW

**Review by ezman — APPROVED** [review]
LGTM

File: environments/xhibit-portal.json [hunk 1]
```diff
@@ -10,6 +10,15 @@
         }
       ]
     },
+    {
```
Comment (davidkelliott) [inline environments/xhibit-portal.json:13]: This file should be in a separate PR and merged in first to ensure the AWS account is created before any networking takes place.

File: policies/networking/expected.rego [hunk 2]
```diff
@@ -13,7 +13,8 @@ expected =
     "cica-development": "10.233.2.64/26",
     "hmpps-production": "10.233.2.128/26",
     "hmcts-development": "10.233.2.192/26",
-    "hmcts-production": "10.233.3.0/26"
+    "hmcts-production": "10.233.3.0/26",
```
Comment (davidkelliott) [inline policies/networking/expected.rego:16]: Actually these 2 blocks and associated tests should have been removed, we now allocate transit gateway and protected subnets from the general subnet set, we used to have to do this manually hence the need to check them, but this is no longer required.

**Review by davidkelliott — CHANGES_REQUESTED** [review]

File: policies/networking/expected.rego (hunk 2 above)
Comment (dms1981) [reply policies/networking/expected.rego]: Those parts have been removed from the latest commit - thanks for the comment!

File: environments/xhibit-portal.json (hunk 1 above)
Comment (dms1981) [reply environments/xhibit-portal.json:13]: That's also been cleaned up as it's not relevant for this stage - thanks for the comment!

**Review by davidkelliott — APPROVED** [review]
👍
